Update applications-databases-postgresql (#2703)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-postgresql.md

> Si la case **Découverte** est cochée, cela signifie qu'une règle de découverte de service existe pour ce service.

</TabItem>
</Tabs>

### Règles de découverte

#### Découverte de service

| Nom de la règle                | Description                                                  |
|:-------------------------------|:-------------------------------------------------------------|
| App-DB-Postgres-Databases-Size | Découvrir les bases de données pour en superviser la taille. |

Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/services-discovery)
pour en savoir plus sur la découverte automatique de services et sa [planification](/docs/monitoring/discovery/services-discovery/#règles-de-découverte).

### Métriques & statuts collectés

Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.

<Tabs groupId="sync">
<TabItem value="Bloat" label="Bloat">

| Métrique                                             | Unité |
|:-----------------------------------------------------|:------|

> | *databases*~*tables*#table.space.usage.bytes         | B     |

Reviewer: db_name et table_name c'est plus clair

> | *databases*~*tables*#table.space.usage.bytes         | B     |
> | *databases*~*tables*#table.space.free.bytes          | B     |
> | *databases*~*tables*#table.dead\_tuple.bytes         | B     |
> | *databases*~*indexes*#index.space.usage.bytes        | B     |
> | *databases*~*indexes*#index.leaf\_density.percentage | %     |

Author: ```suggestion
| *db_name~table_name*#table.space.usage.bytes         | B     |
| *db_name~table_name*#table.space.free.bytes          | B     |
| *db_name~table_name*#table.dead\_tuple.bytes         | B     |
| *db_name~index_name*#index.space.usage.bytes        | B     |
| *db_name~index_name*#index.leaf\_density.percentage | %     |
```

### Utiliser un modèle d'hôte issu du connecteur

1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
2. Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre ressource.
3. Appliquez le modèle d'hôte **App-DB-Postgres-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.

| Macro                | Description                                                                                           | Valeur par défaut | Obligatoire |
|:---------------------|:------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| POSTGRESUSERNAME     | User name used to connect to the database                                                             |                   |      X      |
| POSTGRESPASSWORD     | Password for the defined user name                                                                    |                   |      X      |

> | POSTGRESPORT         | Database Server Port                                                                                  | 5432              |             |

Reviewer: ```suggestion
| POSTGRESPORT         | Database server port                                                                                  | 5432              |             |
```

> | POSTGRESDATABASE     | Database Name                                                                                         | postgres          |             |

Reviewer: ```suggestion
| POSTGRESDATABASE     | Database name                                                                                         | postgres          |             |
```

| CRITICALTABLEFREE        | Thresholds.                                                                                         |                   |             |
| WARNINGTABLEUSAGE        | Thresholds.                                                                                         |                   |             |
| CRITICALTABLEUSAGE       | Thresholds.                                                                                         |                   |             |
| EXTRAOPTIONS             | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose         |             |

</TabItem>
<TabItem value="Cache-Hitratio" label="Cache-Hitratio">

| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|

> | FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |

Reviewer: filter what?

Author: database names ?

> | FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |

Author: ```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

|:-------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| WARNING      | Warning threshold in milliseconds                                                                   |                   |             |
| CRITICAL     | Critical threshold in milliseconds                                                                  |                   |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
<TabItem value="Connection-Number" label="Connection-Number">

| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|

> | FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |

Reviewer: filter what?

> | FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |

Author: ```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

| CRITICAL     | Critical threshold in percent                                                                       | 95                                    |             |
| WARNING      | Warning threshold in percent                                                                        | 90                                    |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose                             |             |

</TabItem>
<TabItem value="Database-Size" label="Database-Size">

| Macro          | Description                                                                                         | Valeur par défaut | Obligatoire |
|:---------------|:----------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERDATABASE | Filter database to checks (Can use regexp)                                                          | .*                |             |

Reviewer: ```suggestion
| FILTERDATABASE | Filter database to checks (can use regexp)                                                          | .*                |             |
```

Reviewer: ?

| WARNINGSIZE    | Warning threshold in bytes, maximum size allowed                                                    |                   |             |
| CRITICALSIZE   | Critical threshold in bytes, maximum size allowed                                                   |                   |             |
| EXTRAOPTIONS   | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
<TabItem value="Locks" label="Locks">

| Macro        | Description                                                                                                                       | Valeur par défaut                     | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|

> | FILTER       | Filter databases                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |

Reviewer: filter what?

> | FILTER       | Filter databases                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |

Author: ```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                                                  | ^(?!(postgres\|template1\|template0)) |             |
```

| CRITICAL     | Critical threshold. (example: "total=250,waiting=5,exclusive=20") 'total', 'waiting', or the name of a lock type used by Postgres | total=250,waiting=5,exclusive=20      |             |
| WARNING      | Warning threshold. (example: "total=250,waiting=5,exclusive=20") 'total', 'waiting', or the name of a lock type used by Postgres  | total=200                             |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                               | --verbose                             |             |

</TabItem>
<TabItem value="Query-Time" label="Query-Time">

| Macro        | Description                                                                                         | Valeur par défaut                     | Obligatoire |
|:-------------|:----------------------------------------------------------------------------------------------------|:--------------------------------------|:-----------:|

> | FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |

Reviewer: filter what?

> | FILTER       | Filter databases                                                                                    | ^(?!(postgres\|template1\|template0)) |             |

Author: ```suggestion
| FILTER       | Filter databases by name (can be a regexp).                                                                                    | ^(?!(postgres\|template1\|template0)) |             |
```

| SQLSTATEMENT | SQL statement that returns a number                                                                 |                   | X           |
| WARNING      | Thresholds.                                                                                         |                   |             |
| CRITICAL     |  Thresholds.                                                                                        |                   |             |
| EXTRAOPTIONS | Any extra option you may want to add to the command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
<TabItem value="Statistics" label="Statistics">

| Macro               | Description                                                                                         | Valeur par défaut                      | Obligatoire |
|:--------------------|:----------------------------------------------------------------------------------------------------|:---------------------------------------|:-----------:|

> | FILTER              | Filter database (can be a regexp)                                                                   | ^(?!(postgres\|template1\|template0)$) |             |

Reviewer: filter what?

> | FILTER              | Filter database (can be a regexp)                                                                   | ^(?!(postgres\|template1\|template0)$) |             |

Author: ```suggestion
| FILTER              | Filter databases by name (can be a regexp).                                                                   | ^(?!(postgres\|template1\|template0)$) |             |
```
